mage/query-modules/python/export-util.md

```diff
@@ -306,8 +383,8 @@ You can create a simple graph database by running the following queries:
 ```cypher
 CREATE (StrangerThings:TVShow {title:'Stranger Things', released:2016, program_creators:['Matt Duffer', 'Ross Duffer']})
 CREATE (Eleven:Character {name:'Eleven', portrayed_by:'Millie Bobby Brown'})
-CREATE (JoyceByers:Character {name:'Joyce Byers', portrayed_by:'Millie Bobby Brown'})
-CREATE (JimHopper:Character {name:'Jim Hopper', portrayed_by:'Millie Bobby Brown'})
+CREATE (JoyceByers:Character {name:'Joyce Byers', portrayed_by:'Winona Ryder'})
+CREATE (JimHopper:Character {name:'Jim Hopper', portrayed_by:'David Harbour'})
```

Comment:
I noticed this once and I was like... will i fix this... is this worth opening a PR for :D :D naaaah... :D thnx